src/module-path.ts

```diff
@@ -1,88 +1,28 @@
 import { IHost, DefaultHost } from './host';
+import * as fs from 'fs';
+import * as browserResolve from 'browser-resolve';
```

Comment:
I would prefer if you use single imports instead of namespace imports.
E.g. `import {sync as browserResolve} from 'browser-resolve';`